src/main/scripts/org/reaktivity/specification/nukleus/amqp/streams/link/attach.as.receiver.only/client.rpt

```diff
@@ -0,0 +1,35 @@
+#
```

Comment:
Script looks good, but concepts are high level now, so name of scenario should be a bit more reaktivity-major, with amqp concepts to describe extensions.

Suggest something like `org/reaktivity/specification/nukleus/amqp/streams/connect.as.receiver.only`
Then others might be:
`connect.as.sender.only`
`connect.as.receiver.then.sender` where `AmpqDataEx` conveys that the data flow bidirectional
`connect.as.sender.then.receiver` where `AmpqDataEx` conveys that the data flow bidirectional